### PR TITLE
feat(channels): new channel detail view

### LINKED ISSUE #1427: Implement new channel detailed view
Implement the newly designed channel management (card) list view.

Design can be found here: https://app.zeplin.io/project/5bacdbad33af886d419f1b34/screen/5c41fa26f65f093797973dbb

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Implement new channel detail view

## Motivation and Context:

Resolves #1427

## How Has This Been Tested?

Manually

## Screenshots (if appropriate):

image

## Types of changes:

New feature

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

**Comment by mrfelton** [comment]
Thanks @JimmyMow . I have made those updates.

File: app/components/Channels/ChannelSummaryListItem.js [hunk 1]
```diff
@@ -8,60 +8,56 @@ import withEllipsis from 'components/withEllipsis'
...
         />
 
         <Flex width={2 / 20}>
-          {channelId && (
-            <ChannelMoreButton onClick={() => showChannelDetail(channelId)} ml="auto" my="auto" />
-          )}
+          <ChannelMoreButton
+            onClick={() => setSelectedChannel(channel_point)}
+            ml="auto"
+            my="auto"
```
Comment (korhaliv) [inline app/components/Channels/ChannelSummaryListItem.js]: Can the same effect be achieved with `Flex` props?

File: app/components/Channels/ChannelCardListItem.js [hunk 2]
```diff
@@ -2,116 +2,89 @@ import React from 'react'
...
-        <Bar opacity={0.3} />
-        <DataRow left={intl.formatMessage({ ...messages.num_updates })} right={numUpdates} py={2} />
-        <Bar opacity={0.3} />
-        <DataRow left={intl.formatMessage({ ...messages.cvs_delay })} right={csvDelay} py={2} />
-      </Box>
+      <ChannelData
+        channel={channel}
+        currencyName={currencyName}
+        networkInfo={networkInfo}
+        s="section"
```
Comment (korhaliv) [inline app/components/Channels/ChannelCardListItem.js]: You probably meant `as`

File: app/components/Channels/ChannelData.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  const {
+    channel_point,
+    commit_fee,
+    fee_per_kw,
+    total_satoshis_sent,
+    total_satoshis_received,
+    csv_delay,
+    num_updates
+  } = channel
+  const fundingTxid = channel_point.split(':')[0]
```
Comment (korhaliv) [inline app/components/Channels/ChannelData.js]: Would be a little more cleaner using destructuring 
`const [fundingTxid] = channel_point.split(':')`

Comment (mrfelton) [reply app/components/Channels/ChannelData.js]: Updated.

File: app/components/Channels/ChannelCardListItem.js (hunk 2 above)
Comment (mrfelton) [reply app/components/Channels/ChannelCardListItem.js]: Updated.

File: app/components/Channels/ChannelSummaryListItem.js (hunk 1 above)
Comment (mrfelton) [reply app/components/Channels/ChannelSummaryListItem.js]: Yes. Updated.

**Comment by mrfelton** [comment]
Have made those updates, thanks @korhaliv

**Comment by JimmyMow** [comment]
Copy changes look good @mrfelton. Approved from me

**Pull request merged**